spotify-track-search.el

```diff
@@ -24,6 +24,20 @@
   "Major mode for displaying the track listing returned by a Spotify search.")
 
 (defun spotify-track-select ()
+  "Plays the track, album or artist under the cursor. If the cursor is on a
+button representing an artist or album, start playing that artist or album.
+Otherwise, play the track selected."
+  (interactive)
+  (let ((button-selected (spotify-track-selected-button-type)))
+    (cond ((string= "Artist" button-selected)
+	   (spotify-track-artist-select))
+	  ((string= "Album" button-selected)
+	   (spotify-track-album-select))
+	  (t (spotify-track-select-default)))
+    )
+  )
```

Comment:
I'd move these closing parenthesis to the end of the `(t (spotify-track-select-default)))`, as this is how the current code is styled.

This comment applies to other similar snippets of code.

Reply:
Done.